src/mgmt/cli_debug.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * (C) Copyright 2016-2020 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+/**
+ * \file
+ *
+ * debug apis for DAOS  client library. It exports the debug APIs defined in
+ * cli_debug.h
+ */
+
+#include <daos/mgmt.h>
+#include <daos/sys_debug.h>
+
+#include <daos/agent.h>
+#include <daos/drpc_modules.h>
+#include <daos/drpc.pb-c.h>
+#include <daos/event.h>
+#include <daos/job.h>
+#include "rpc.h"
+#include <errno.h>
+
+int
+dc_debug_set_params(tse_task_t *task)
+{
+	daos_set_params_t		*args;
+	struct cp_arg			cp_arg;
+	struct mgmt_params_set_in	*in;
+	crt_endpoint_t			ep;
+	crt_rpc_t			*rpc = NULL;
+	crt_opcode_t			opc;
+	int				rc;
+
+	args = dc_task_get_args(task);
+	rc = dc_mgmt_sys_attach(args->grp, &cp_arg.sys);
+	if (rc != 0) {
+		D_ERROR("failed to attach to grp %s, rc "DF_RC".\n", args->grp,
```

Comment:
No . on the end here after DF_RC.  Also applies elsewhere in this PR.
```suggestion
		D_ERROR("failed to attach to grp %s, rc "DF_RC"\n", args->grp,
```

Reply:
Done.